src/views/__snapshots__/AppShell.spec.ts.snap

```diff
@@ -100,15 +100,18 @@ exports[`AppShell.vue renders component with route 1`] = `
             <div
               class="is-active is-menu-item nav-item"
               data-testid="home"
-              iconcustom=""
+              iconcustom="[object Object]"
               subnav="[object Object]"
             >
               <a>
                 <div
                   class="nav-icon"
                 >
                   
-                  <!--v-if-->
+                  <!-- eslint-disable-next-line vue/no-v-html -->
+                  <div>
+                    [object Object]
```

Comment:
same as the one above somewhere

Reply:
Deferring this conversation to https://github.com/kumahq/kuma-gui/pull/377#discussion_r966842541.